Solution for budget-app exercise

fesiddi opened the pull request:
This time i don't see the other commit so i think it's right...

storrisi reviewed (changes requested):
As well for the other exercise, your code is not usable.

For example, the function create_spend_chart is defined but never called.

I suggest you to add a real case use of your implementation

fesiddi commented:
Hi Simone i've modified the solution file location to 'solution'/federicosiddi/budget.py' because otherwise the original file of the upstream repo will be modified. I've added the test code inside my solution file so it can be tested.
